Tests/FluentAssertions.Specs/Primitives/StringAssertionSpecs.BeEquivalentTo.cs

```diff
@@ -12,6 +13,33 @@ public partial class StringAssertionSpecs
 {
     public class BeEquivalentTo
     {
+        [Fact]
+        public void Use_custom_comparer()
```

Comment:
🤔 I think this test needs a better name

Reply:
I renamed the two tests to `Succeed_for_different_strings_using_custom_matching_comparer` and `Fail_for_same_strings_using_custom_not_matching_comparer`